dash/testing/application_runners.py

```diff
@@ -206,6 +207,59 @@ def stop(self):
         self.started = False
 
 
+class MultiProcessRunner(BaseDashRunner):
+    def __init__(self, keep_open=False, stop_timeout=3):
+        super().__init__(keep_open, stop_timeout)
+        self.proc = None
+
+    # pylint: disable=arguments-differ
+    def start(self, app, start_timeout=3, **kwargs):
+        self.port = kwargs.get('port', 8050)
+
+        def target():
+            app.scripts.config.serve_locally = True
+            app.css.config.serve_locally = True
+
+            options = kwargs.copy()
+
+            try:
+                app.run(threaded=True, **options)
+            except SystemExit:
+                logger.info("Server stopped")
+                raise
+            except Exception as error:
+                logger.exception(error)
+                raise error
+
+        # pylint: disable=import-outside-toplevel,no-member
+        import multiprocess as mp
```

Comment:
Why does this import need to be inside here?

Reply:
Moved :ok_hand: